### ISSUE TITLE
WHen NVDA is started from source with python.exe it is not possible to interact with any console window.

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#8653]
### Steps to reproduce:
1. On system with all NVDA dependencies installed clone NVDA git repository and run scons source.
2. move to the source directory and type python nvda.pyw
3. Try to read a console window in which NVDA is started.
4. Open another cmd and try to interact with it.
### Actual behavior:
Screen reviev keys didn't work and the following is written in the log each time when the console is focused.
ERROR - NVDAObjects.behaviors.LiveText._monitor (14:54:56.263):
Error getting initial lines
Traceback (most recent call last):
  File "NVDAObjects\behaviors.py", line 260, in _monitor
    oldLines = self._getTextLines()
  File "NVDAObjects\window\winConsole.py", line 49, in _getTextLines
    return winConsoleHandler.getConsoleVisibleLines()
  File "winConsoleHandler.py", line 121, in getConsoleVisibleLines
    consoleScreenBufferInfo=wincon.GetConsoleScreenBufferInfo(consoleOutputHandle)
  File "wincon.py", line 70, in GetConsoleScreenBufferInfo
    raise WinError()
WindowsError: [Error 6] Nieprawidłowe dojście. which in polish mean something like the handle is invalid.

### Expected behavior:
NVDA should be able to read a console window.
#### NVDA Installed/portable/running from source:
Running from source
#### NVDA version:
master commit ec3eeee941f6b47f6ebb8ead9dc75bdc3efddfa1 (HEAD -> master, nvaccess/master
#### Windows version:
Windows 7 x64
#### Name and version of other software in use when reproducing the issue:
Python 2.7.15, Visual Studio community 2017
#### Other information about your system:
None
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by leonardder** [comment]
You are expected to start nvda with pythonw, not with python. Does the problem persist if you do this?

**Comment by lukaszgo1** [comment]
When starting NVDA with pythonw console windows can be read. I am aware of the fact that starting with pythonw is a recomended way, however as mentioned in the readme in rare occasion using python.exe to start might be useful.

**Comment by codeofdusk** [comment]
Fixed in #9771.

**Comment by CyrilleB79** [comment]
Tested with `pythonw` or `pyw -3-32`. All is working correctly.

With `python` or `py -3-32` on windows 10 1709, @codeofdusk wrote in #10352 (duplicate of this one):

> On 1709, you can get some limited console support when running from source by changing "Windows Console support" to "prefer UIA" in advanced preferences. Caret movement and review by granularity other than line has limited to no support 

Tested and confirmed to work on my side.

**Comment by Adriani90** [comment]
@lukaszgo1 if this issue is not reproducible for you in last NVDA Alpha, I think we can Close this issue as works for me.

**Comment by codeofdusk** [comment]
Well, now make that fixed in #10964 (#9771 was reverted).
